### ISSUE TITLE
Implement traits from the `rand` crate.

### ISSUE THREAD

**Issue by Lemmih** [reem/rust-ordered-float#83]
It would be nice to have these implementations behind an optional dependency on `rand`. Would you accept a PR?

```rust
impl<T> Distribution<NonNan<T>> for Standard {}
impl<T> Distribution<NonNan<T>> for Open01 {}
impl<T> Distribution<NonNan<T>> for OpenClosed01 {}
impl<T> Distribution<OrderedFloat<T>> for Standard {}
impl<T> Distribution<OrderedFloat<T>> for Open01 {}
impl<T> Distribution<OrderedFloat<T>> for OpenClosed01 {}
```

**Comment by mbrubeck** [comment]
Yes, a PR would be welcome.

**Comment by mbrubeck** [comment]
Fixed by #84.
